### PR TITLE
Update grpc bbe contents

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/3804

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/grpc-client-client-streaming/grpc_client_streaming.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # gRPC client - Client-side streaming RPC
 
-The gRPC Server Connector is used to expose gRPC services over HTTP/2. In a gRPC client streaming scenario, the client writes a sequence of messages and sends them to the server. Once the client has finished writing the messages, it waits for the server to read them and return a response.
+The `grpc:Client` communicates with a gRPC server over HTTP2. A `grpc:Client` is created by providing the endpoint url of gRPC server. In the client streaming scenario, once connected, the client sends a sequence of messages to the remote service and waits for the server to read them and return a single response. Use this to send multiple requests and get a single response back.
```
Comment (shafreenAnfar) [inline examples/grpc-client-client-streaming/grpc_client_streaming.md:3]: endpoint URL

File: examples/grpc-client-server-streaming/grpc_server_streaming.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # gRPC client - Server-side streaming RPC
 
-The gRPC Server Connector is used to expose gRPC services over HTTP/2. In a gRPC server streaming scenario, a client sends a request to the server and gets a stream to read the messages until all the messages are read.
+The `grpc:Client` communicates with a gRPC server over HTTP2. A `grpc:Client` is created by providing the endpoint url of gRPC server. In the server streaming scenario, once connected, the client sends a request to the remote service and gets a message stream as the response which contains multiple messages. Use this to send a single request and get multiple responses back.
```
Comment (shafreenAnfar) [inline examples/grpc-client-server-streaming/grpc_server_streaming.md:3]: ditto

Comment (dilanSachi) [reply examples/grpc-client-server-streaming/grpc_server_streaming.md]: Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/3785/commits/58b1668c71f383fe61fcd9fd3f59ded060eee2fd

File: examples/grpc-client-client-streaming/grpc_client_streaming.md (hunk 1 above)
Comment (dilanSachi) [reply examples/grpc-client-client-streaming/grpc_client_streaming.md]: Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/3785/commits/58b1668c71f383fe61fcd9fd3f59ded060eee2fd

File: examples/grpc-client-bidirectional-streaming/grpc_bidirectional_streaming.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # gRPC client - Bidirectional streaming RPC
 
-The gRPC server connector exposes the gRPC service over HTTP2. In a gRPC bidirectional streaming scenario, the gRPC service and the client operate when each other sends a sequence of messages using a read-write stream. In such scenarios, the two streams operate independently. Therefore, clients and servers can read and write in any order.
+The `grpc:Client` communicates with a gRPC server over HTTP2. A `grpc:Client` is created by providing the endpoint url of gRPC server. In the bidirectional streaming scenario, Once connected, the client and the service sends message streams to each other. In this scenario, the two streams operate independently and therefore, clients and servers can read and write in any order. Use this to send multiple requests to a server and get multiple responses back.
```
[the remark this answers is not in the document]
Comment (dilanSachi) [reply examples/grpc-client-bidirectional-streaming/grpc_bidirectional_streaming.md]: Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/3785/commits/58b1668c71f383fe61fcd9fd3f59ded060eee2fd

File: examples/grpc-client-bidirectional-streaming/grpc_bidirectional_streaming.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # gRPC client - Bidirectional streaming RPC
 
-The gRPC server connector exposes the gRPC service over HTTP2. In a gRPC bidirectional streaming scenario, the gRPC service and the client operate when each other sends a sequence of messages using a read-write stream. In such scenarios, the two streams operate independently. Therefore, clients and servers can read and write in any order.
+A `grpc:Client` is created by providing the endpoint URL of a gRPC server. In the bidirectional streaming scenario, once connected, the client and the service send message streams to each other. In this scenario, the two streams operate independently, and therefore, clients and servers can read and write in any order. Use this to send multiple requests to a server and get multiple responses back.
```
Comment (shafreenAnfar) [inline examples/grpc-client-bidirectional-streaming/grpc_bidirectional_streaming.md:3]: to send multiple messages to a server and get multiple messages back.

Comment (dilanSachi) [reply examples/grpc-client-bidirectional-streaming/grpc_bidirectional_streaming.md]: Updated

**Review by praneesha — APPROVED** [review]

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**